invenio_app_ils/records/loaders/schemas/preserve_cover_metadata.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Preserve cover metadata."""
+
+from flask import has_request_context
+
+
+def preserve_cover_metadata(data, prev_record):
+    """Preserver cover metadata if they exist."""
+    if not has_request_context() or prev_record is None:
```

Comment:
```suggestion
    if prev_record is None:
```
You don't need the `has_request_context` because you are not using anything from current request